apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js

```diff
@@ -105,7 +108,15 @@ class ReviewApplication extends React.Component {
 
         <ApplicationDetails>
           <UserLink>
-            <img src={applicant.avatar} style={{ width: '32px', height: '32px', marginRight: '10px' }} />
+            <img
+              alt=""
```

Comment:
It's _no_ alt that causes a11y issues. The recommendation is to either set it to something meaningful, or intentionally set as blank if the image is decorative-only. I don't think it would help users of screen readers understand this page to have text here. Do you?